src/templatizer.js

```diff
@@ -4,46 +4,41 @@ const argumentsTemplate = JSON.parse(fs.readFileSync('./src/template/args.json')
 const optionTemplate = require('./template/option.json');
 
 const templatizer = (context, config) => {
-    argumentsTemplate.definitions.arguments.properties = {};
+    argumentsTemplate.definitions.object.properties = {};
     let result = argumentsTemplate;
-    const customPrefix = config.prefixes.custom;
-    const nonFlagPrefix = config.prefixes.nonFlag;
     context.options.map((option) => {
         let optionSchema = Object.assign({}, optionTemplate);
         const argumentName = option.longName ? option.longName :
                 (option.shortName ? option.shortName : '');
         switch (argumentName) {
             case '--version':
-                optionSchema.id = customPrefix + 'version';
                 optionSchema.type = 'null';
+                optionSchema.usage = 'version';
                 break;
             case '--help':
-                optionSchema.id = customPrefix + 'help';
                 optionSchema.type = 'null';
```

Comment:
Why it is a string? Seems like just setting it to `null` will be enough.

Reply:
@xferra in our EP "null" is string
https://github.com/repometric/ep/blob/master/ep/lh/ep-lh-0003.md#arguments